smoke-test/src/main/java/org/opennms/smoketest/utils/RestHealthClient.java

```diff
@@ -74,8 +86,11 @@ private WebTarget getTargetFor(final String path){
     }
 
     public String getProbeHealthResponse(){
+        WebTarget webTarget = getTargetFor(PROBE);
+        boolean isAvailable = checkUriAvailability(webTarget.getUri());
+        if (isAvailable) {
```

Comment:
I do not understand why this is needed. There are checks to retry on exception in all callers of this method.
If there is an exception happening that is not covered/ignored in these handlers, we should examine why this happens and maybe add special handling there.

Reply:
This extra check will avoid a ConnectionReset exception to be thrown while initializing the docker containers: the webtarget.request tries to get a response from a server (docker container) not ready/fully created yet. Result is a lof of spam in the logs. Where are these checks done? Maybe we can add this exception there

Reply:
`waitUntilReadyWrapped` in `OpenNMSContainer`, `MinionContainer` and `SentinelContainer`.